testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -329,6 +330,7 @@
 # Contains one of each residue in 'nucleic' selections
 NUCLsel = resource_filename(__name__, 'data/nucl_res.pdb')
 
+RANDOM_WALK = resource_filename(__name__, 'data/xyz_random_walk.xyz')
```

Comment:
If you use xtc the filesize will be smaller. You can change this in a rebase to remove the xyz from the history in git.